parser/influxdb.go

```diff
@@ -142,11 +142,11 @@ func (line *InfluxDBLineProtocol) ParseLine(s string) error {
 		tag := strings.Trim(tagScanner.Text(), "\u0000")
 		tagValue := strings.Split(tag, "=")
 
-		if len(tagValue) != 2 {
+		if len(tagValue) < 2 {
 			break
 		}
 
-		line.tags[tagValue[0]] = tagValue[1]
+		line.tags[tagValue[0]] = strings.Join(tagValue[1:], "=")
```

Comment:
This can really be done with strings.SplitN(tag,"=",2), which will return two substrings, where ` the last substring will be the unsplit remainder.`.

This is considerably faster, since all processing will stop at the first =, and no join is needed. Whether we will notice the difference or not is a different matter.

Reply:
I'll check out SplitN.

Join will only be a performance impact if the remaining number of strings is > 1. Otherwise Join will simply return the first string in the list. Sure, it'll still do a comparison for list length, but I would've done that in our code myself. No comparisons sounds like would be even better, though, and string-building might be slow.